.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -38,12 +38,12 @@ body:
 
           - text models: @ArthurZucker
           - vision models: @amyeroberts, @qubvel
-          - speech models: @ylacombe, @eustlb
+          - speech models: @eustlb
           - graph models: @clefourrier
 
         Library:
 
-          - flax: @sanchit-gandhi
+          - flax: @gante and @Rocketknight1
```

Comment:
Two notes:
- Do we want to update flax, or remove flax/tensorflow?
- If we don't remove, these two seem the most appropriate (it's XLA under the hood).